styles.css

```diff
@@ -169,24 +169,22 @@ body {
 /* Services Section */
 
 h1{
```

Comment:
I'll create a class for that because it'll change the style for the existing h1.

Reply:
Thank you ✨